### PR TITLE
Use common include directory for all ports

### PR DESCRIPTION
**Pull request by sbc100**
We already use a common "lib" directory to install al the libs.  This
change simply installs headers in the same common "sysroot".

This makes the ports install more like normall "make install" scripts
would.

The only port I didn't include here was cocos2d because it seemed a
little overwhelming.

### CODE REVIEW

File: embuilder.py [hunk 1]
```diff
@@ -125,28 +111,22 @@ def build(src, result_libs, args=[]):
...
-  parser.add_argument('--help', '-h', action='store_true', help='print help message')
+  parser = argparse.ArgumentParser(description=__doc__, usage=get_usage())
   parser.add_argument('--lto', action='store_true', help='build bitcode object for LTO')
   parser.add_argument('--pic', action='store_true',
                       help='build relocatable objects for suitable for dynamic linking')
   parser.add_argument('--force', action='store_true',
                       help='force rebuild of target (by removing it first)')
-  parser.add_argument('operation')
-  parser.add_argument('targets', nargs='+')
+  parser.add_argument('operation', help='currnetly only "build" is supported')
```
Comment (kripken) [inline embuilder.py]: typo in `currently`

File: tests/test_other.py [hunk 2]
```diff
@@ -798,7 +798,10 @@ def verify_includes(stderr):
       includes = stderr[start:end]
       includes = [i.strip() for i in includes.splitlines()[1:-1]]
       for i in includes:
-        self.assertContained(path_from_root('system'), i)
+        if shared.Cache.dirname in i:
```
Comment (kripken) [inline tests/test_other.py]: this isn't clear to me. perhaps a comment?

Comment (sbc100) [reply tests/test_other.py]: Sorry this isn't best test in the world.  Adding a comment.

**Comment by sbc100** [comment]
Added a ChangeLog entry since this change could be user visible in some cases.

**Pull request merged**